plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -2750,22 +2786,20 @@ public void testLocalDynamicFilteringWithSelectiveBuildSizeJoin()
                 result.getQueryId(),
                 new QualifiedObjectName(ICEBERG_CATALOG, "tpch", "lineitem"));
 
-        // Assert no split level pruning occurs. If this starts failing a new totalprice may need to be selected
-        assertThat(probeStats.getTotalDrivers()).isEqualTo(numberOfFiles);
```

Comment:
Why removed?

Reply:
This was discussed offline and approved by @sopel39 .
This causes test to be flaky in runs we receive 2 or 3 randomly.
With MinIO this was happening more often like 50% of time.

Reply:
Issue is addressed in https://github.com/trinodb/trino/pull/10997